src/QsCompiler/CompilationManager/TypeChecking.cs

```diff
@@ -262,7 +272,15 @@ private static T ContainingParent<T>(Position pos, IReadOnlyCollection<(Position
                 // add all callable declarations
                 var callablesToCompile = AddItems(
                     file.GetCallableDeclarationHeaderItems(),
-                    (pos, name, decl, att, doc) => ContainingParent(pos, namespaces).TryAddCallableDeclaration(file.FileName, new QsLocation(pos, name.Item2), name, Tuple.Create(decl.Item1, decl.Item3), att, decl.Item2, doc),
+                    (pos, name, decl, att, doc) =>
+                        ContainingParent(pos, namespaces).TryAddCallableDeclaration(
+                            file.FileName,
+                            new QsLocation(pos, name.Item2),
+                            name,
+                            Tuple.Create(decl.Item1, decl.Item2.Signature),
+                            att,
+                            decl.Item2.Access.ValueOr(Access.Public),
```

Comment:
If I understand correctly, before this change if the callable did not specify an access modifier keyword, the value of `decl.Item2.AccessModifier` would be `DefaultAccess` which is what would be passed to this method. After this change, `decl.Item2.Access` can be `null` and its is `null` when an access modifier keyword is not specified, correct?

Reply:
Yes, that's right.

Reply:
Thanks!